server/policies/utils.ts

```diff
@@ -3,12 +3,16 @@ import { IncorrectEditionError } from "@server/errors";
 import { User, Team } from "@server/models";
 import Model from "@server/models/base/Model";
 
-export function and(...args: boolean[]) {
-  return args.every(Boolean);
+type Args = boolean | string | Args[];
```

Comment:
All of these changes are to allow policies to return `membershipIds` instead of just booleans